### ISSUE TITLE
i use pytorch1.0.0(cuda 9) show error,what is your env version?

### ISSUE THREAD

**Issue by mathpopo** [kuangliu/pytorch-retinanet#66]
def msr_init(net):
    '''Initialize layer parameters.'''
    for layer in net:
        if type(layer) == nn.Conv2d:
            n = layer.kernel_size[0]*layer.kernel_size[1]*layer.out_channels
            layer.weight.data.normal_(0, math.sqrt(2./n))
            layer.bias.data.zero_()
        elif type(layer) == nn.BatchNorm2d:
            layer.weight.data.fill_(1)
            layer.bias.data.zero_()
        elif type(layer) == nn.Linear:
            layer.bias.data.zero_()

_, term_width = os.popen('stty size', 'r').read().split()-------

os.popen   not enough values to unpack
